### PR TITLE
[Features] Support fp16 models.

### PR DESCRIPTION
**Pull request by zytx121**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Add five fp16 models:
- rotated_retinanet_obb_r50_fpn_fp16_1x_dota_le90.py
- s2anet_r50_fpn_fp16_1x_dota_le135.py
- roi_trans_r50_fpn_fp16_1x_dota_le90.py
- redet_re50_refpn_fp16_1x_dota_le90.py
- oriented_rcnn_r50_fpn_fp16_1x_dota_le90.py

## Modification
Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```
Comment (ZwwWayne) [inline run.sh:1]: This file seems to be unnecessary and should be removed.

Comment (ZwwWayne) [inline run.sh:1]: This file seems to be unnecessary and should be removed.

**Review by yangxue0827 — APPROVED** [review]
LGTM

**Comment by zytx121** [comment]
@ZwwWayne We could merge it now.

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**